apps/web/modules/survey/follow-ups/components/follow-up-modal.tsx

```diff
@@ -317,15 +340,47 @@ export const FollowUpModal = ({
         replyTo: defaultValues?.replyTo ?? [userEmail],
         subject: defaultValues?.subject ?? "Thanks for your answers!",
         body: defaultValues?.body ?? getSurveyFollowUpActionDefaultBody(t),
+        attachResponseData: defaultValues?.attachResponseData ?? false,
       });
     }
-  }, [open, defaultValues, emailSendToOptions, form, userEmail, locale]);
+  }, [open, defaultValues, emailSendToOptions, form, userEmail, locale, t]);
 
   const handleModalClose = () => {
     form.reset();
     setOpen(false);
   };
 
+  const emailSendToQuestionOptions = emailSendToOptions.filter(
+    (option) => option.type === "openTextQuestion" || option.type === "contactInfoQuestion"
+  );
+  const emailSendToHiddenFieldOptions = emailSendToOptions.filter((option) => option.type === "hiddenField");
+  const emailSendToEmailOptions = emailSendToOptions.filter((option) => option.type === "email");
+
+  const renderSelectItem = (option: EmailSendToOption) => {
+    return (
+      <SelectItem key={option.id} value={option.id}>
+        {option.type === "hiddenField" ? (
+          <div className="flex items-center space-x-2">
+            <EyeOffIcon className="h-4 w-4" />
+            <span>{option.label}</span>
+          </div>
+        ) : option.type === "email" ? (
+          <div className="flex items-center space-x-2">
+            <UserIcon className="h-4 w-4" />
+            <span className="overflow-hidden text-ellipsis whitespace-nowrap">{option.label}</span>
+          </div>
+        ) : (
```

Comment:
please also include the user name and `yourself` as mentioned in the ticket

Reply:
done 👍